programmers-coding-test/jaeseong/greedy/숫자카드게임.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+const rl = require("readline-sync");
+let [ n, m ] = rl.question("n m input please : \n>>").split(" ");
+let data = rl.question("array input please : \n>>").split(" ");
```

Comment:
이건 2차원 배열이 아니라 1차원으로만 만들어질텐데...?
```suggestion
const datas = [];
for (let i = 0; i < m; i++ {
  const data = rl.question("array input please : \n>>").split(" ");
  datas.push(data);
}
```